src/core/tools/common/thrift_message_parser.h

```diff
@@ -60,6 +60,10 @@ namespace dsn
     // response format:
     //     <total_len(int32)> <thrift_string> <thrift_message_begin> <body_data(bytes)> <thrift_message_end>
 
+# define THRIFT_HDR_SIG (*(uint32_t*)"THFT")
```

Comment:
There are multiple cases referencing this "THFT" in thrift-message-parser, and a micro helps make sure they are all consistent. For the other message parsers, they are referenced only once.